File: Solutions In Java/p053.java
/* 
 * Solution to Project Euler problem 53
 * Copyright (c) Sankha Saha (CODR). All rights reserved.
 * 
 * https://github.com/codr07/Project-Euler-Solusions
 */

import java.math.BigInteger;


public final class p053 implements EulerSolution {
	
	public static void main(String[] args) {
		System.out.println(new p053().run());
	}
	
	
	public String run() {
		BigInteger MILLION = BigInteger.TEN.pow(6);
		int count = 0;
		for (int n = 1; n <= 100; n++) {
			for (int r = 0; r <= n; r++) {
				if (Library.binomial(n, r).compareTo(MILLION) > 0)
					count++;
			}
		}
		return Integer.toString(count);
	}
	
}

File: Solutions in Python/p053.py
# 
# Solution to Project Euler problem 53
# Copyright (c) Sankha Saha (CODR). All rights reserved.
# 
#
# https://github.com/codr07/Project-Euler-Solusions
# 

import math


def compute():
	ans = sum(1
		for n in range(1, 101)
		for k in range(0, n + 1)
		if math.comb(n, k) > 1000000)
	return str(ans)


if __name__ == "__main__":
	print(compute())
